week-1/Homework/extra/2-piping.js

```diff
@@ -16,43 +16,53 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  const total = a + b;
+  return Number(total.toFixed(2));
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  const total = a * b;
+  return Number(total.toFixed(2));
 }
 
-function format() {
-
+function format(a) {
+  const formated = "£" + a;
+  return formated;
 }
 
-const startingValue = 2
+const startingValue = 2;
+
+let finalResult = format(multiply(add(startingValue, 10), 2));
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode = 
+// All the code is written on one line, more difficult to read and modify.
+
+let badCode = finalResult;
```

Comment:
You applied the correct "bad code"! But how come you decided to use a different variable for the result? (i.e. `finalResult` instead of `badCode`)?

Reply:
lol I am not sure why I did it, I removed it now.

Reply:
I am not sure why I added another variable lol. I removed it now.